package.json

```diff
@@ -13,8 +13,8 @@
     "lint:eslint": "eslint --config .eslintrc.js \"src/\" \"e2e/\"",
     "lint:prettier": "prettier \"{src,e2e}/**/*.{ts,tsx,js,jsx,css}\" \"**/*.{md,mdx,yml}\"",
     "test": "jest src",
-    "e2e-test-node": "jest --config=jest.e2e.config.js e2e/node",
-    "e2e-test-browser": "echo \"This project has no e2e browser tests\"",
+    "test:e2e:node": "jest --config=jest.e2e.config.js e2e/node",
```

Comment:
I think you can also merge `jest.e2e.config.js` with `jest.config.js` — see: https://github.com/inrupt/solid-client-authn-js/blob/main/jest.config.ts for how it's done.